apps/web-twig-demo/assets/scripts/toast-dynamic.ts

```diff
@@ -39,7 +41,7 @@ export const clearQueue = (event, containerId) => {
 };
 
 // Make functions available in the global scope
-window.addDynamicToast = addDynamicToast;
-window.clearQueue = clearQueue;
+(window as any).addDynamicToast = addDynamicToast;
```

Comment:
Btw, you can do something like 

```ts
declare global {
    interface Window { addDynamicToast: () => void; }
}
```

Reply:
thank you 👍  changed to use this method, its better that `any`